src/cascade/model/session.py

```diff
@@ -316,6 +344,36 @@ def _check_vars(var):
             for key, one_var in group.items():
                 one_var.check(f"{group_name}-{key}")
 
+    @staticmethod
+    def _point_age_time_to_interval(data):
+        for at in ["age", "time"]:  # Convert from point ages and times.
+            for lu in ["lower", "upper"]:
+                if f"{at}_{lu}" not in data.columns and at in data.columns:
+                    data = data.assign(**{f"{at}_{lu}": data[at]})
+        return data.drop(columns={"age", "time"} & set(data.columns))
+
+    @staticmethod
+    def _amend_data_input(data):
+        """If the data comes in without optional entries, add them.
+        This doesn't translate to internal IDs for Dismod-AT. It rectifies
+        the input, and this is how it should be saved or passed to another tool.
+        """
```

Comment:
This function isn't changing things to the Dismod-AT data representation. It's allowing the user-specified data to be simpler and then augmenting it to be a full specification. It's a layer of validation. The fully-specified version will then be stored in the result object.